packages/edit-post/src/editor.js

```diff
@@ -91,6 +91,7 @@ class Editor extends Component {
 					settings={ editorSettings }
 					post={ post }
 					initialEdits={ initialEdits }
+					useSubRegistry={ false }
```

Comment:
I feel this prop should be passed down from `editor/provider` to `block-editor/provider` as well right (unless we access the `block-editor` store from the `editor` components (which we probably do on a second thought but maybe we shouldn't) ?

Reply:
> I feel this prop should be passed down from `editor/provider` to `block-editor/provider` as well right (unless we access the `block-editor` store from the `editor` components (which we probably do on a second thought but maybe we shouldn't) ?

Right now we just explicitly pass `false`:

https://github.com/WordPress/gutenberg/blob/42b27486c11c2dd937b81234e3db27169e53e6e5/packages/editor/src/components/provider/index.js#L159

Which I think is how we want it to be:

- If you wanted your own block editor with its own subregistry, you'd render BlockEditorProvider yourself
- If you wanted your own _post_ editor with its own subregistry, you'd render EditorProvider which would create a subregistry. The EditorProvider would pass `useSubRegistry={ false }` to its BlockEditorProvider, but that block editor still operates in the same subregistry as created for the EditorProvider

Reply:
Makes sense, thanks for the explanation.